plugins/pixart-rf/fu-pxi-device-ota.h

```diff
@@ -0,0 +1,187 @@
+
+
+#pragma once
+
+
+#include <glib.h>
+
+
+#define CMD_FW_OTA_INIT 0x10
+#define CMD_FW_WRITE    0x17
+#define CMD_FW_UPGRADE  0x18
+#define CMD_FW_MCU_RESET    0x22    /* Todo: remove this OP code handle after windows OTA app ready */
+#define CMD_FW_GET_INFO     0x23
+
+#define CMD_FW_OBJECT_CREATE 0x25
+#define CMD_FW_OTA_INIT_NEW 0x27
+#define CMD_FW_OTA_RETRANSMIT 0x28
+#define CMD_FW_OTA_DISCONNECT 0x29
+
+#define ERR_COMMAND_SUCCESS           0x00
+#define ERR_COMMAND_UPDATE_FAIL       0xFF
+#define EVT_COMMAND_COMPLETE        (0x0E)
+#define CMD_COMPLETE_HDR_SZ     1
+
+#define OTA_BUFFER_SIZE 256
+#define MAX_OBJECT_SIZE 4096
+#define PXI_OTA_PAYLOAD 20
+#define PXI_OTA_BUF_SZ 32
+
+#pragma pack(1)

```

Comment:
don't use this, please use `typedef struct __attribute__((packed)) {` on structs